Venus-venuslangmuir/Homework-4/madlib-with-loops/js/app.js

```diff
@@ -0,0 +1,62 @@
+var startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+var startupY = ['Slack', 'Trello', 'Tesla', 'Hyperloop', 'Harvest'];
+
+// Get the h1 with the id of 'xForY'
+const h1 = document.querySelector('#xForY');
+// Get the 'Create New Startup' button
+const createBtn = document.querySelector('#create');
+
+// Add a click event to the create button to generate the random message on the page
+createBtn.addEventListener('click', function () {  
+    var random1 = Math.floor((Math.random() * startupX.length));
```

Comment:
It doesn't break anything but generally its good practive to try and keep your variables consistent such as using all var or  all let and const